src/System.Drawing.Common/tests/Printing/PrinterSettingsTests.cs

```diff
@@ -384,21 +377,6 @@ public void PrinterResolutions_ReturnsExpected()
             Assert.NotNull(printerSettings.PrinterResolutions);
         }
 
-        public static IEnumerable<object[]> IsDirectPrintingSupported_ImageFormatSupported_TestData()
-        {
-            yield return new object[] { ImageFormat.Jpeg };
-            yield return new object[] { ImageFormat.Png };
-        }
-
-        [ActiveIssue(20884, TestPlatforms.AnyUnix)]
-        [ConditionalTheory(Helpers.AnyInstalledPrinters, Helpers.GdiplusIsAvailable)]
-        [MemberData(nameof(IsDirectPrintingSupported_ImageFormatSupported_TestData))]
-        public void IsDirectPrintingSupported_ImageFormatSupported_ReturnsExpected(ImageFormat imageFormat)
-        {
-            var printerSettings = new PrinterSettings();
-            Assert.Equal(true, printerSettings.IsDirectPrintingSupported(imageFormat));
```

Comment:
Should we have a tests for these that run the method and verify that it does crash, but do not check the result?

Reply:
Sure, will change.